src/lib/game-saving/account.ts

```diff
@@ -337,4 +340,19 @@ export const getSnapshotData = async (id: string): Promise<SnapshotData | null>
 
 export const updateUserGitHubToken = async (userId: string, githubAccessToken: string, githubId: string, githubUsername: string): Promise<void> => {
     await updateDocument('users', userId, { githubAccessToken, githubId, githubUsername });
+}
+
+async function hashCodeToBigInt(string : string) : Promise<bigint>{
+	return BigInt(`0x${ await sha256Hash(string)}`);
```

Comment:
usage of a BigInt isn't necessary here - can we just use String's hashCode method, which comes by default?

Reply:
can you point me to the docs for that? I honestly don't think that exists in js, only Java.